docker/Dockerfile

```diff
@@ -0,0 +1,67 @@
+ARG CCF_VERSION="6.0.1"
+
+# Use fixed image version to ensure reproducibility
+ARG BASE_IMAGE=mcr.microsoft.com/azurelinux/base/core:3.0.20250402
+
+FROM ${BASE_IMAGE} AS builder
+
+ARG CCF_VERSION
+ARG CCF_PLATFORM
+ARG SCITT_VERSION_OVERRIDE
+
+RUN tdnf install -y ca-certificates && \
```

Comment:
please use specific versions to aid in reproducibility

Reply:
Addressed in https://github.com/microsoft/scitt-ccf-ledger/pull/298/commits/bf22322a96a07fda8eae637247b494145222c10c